### PR TITLE
adding the image to the donation view as an image

### PR DESCRIPTION
**Pull request by alejandroGonGon**
About :

This Pull Request add an image to the Donation View in the Admin Panel, insted of being an URL to be clicked there is an Image in the information.

Extra :

I also add the same logic to the User section in the Donation View.

Demo: 
image
image

### CODE REVIEW

**Comment by LautaroPardo0210** [comment]
Task linked: [CU-1tkgbhf Show donation image URL as clickable](https://app.clickup.com/t/1tkgbhf)

File: app/Orchid/Screens/DonationEditScreen.php [hunk 1]
```diff
@@ -86,6 +86,11 @@ public function layout(): array
     {
         return [
              Layout::legend('donation', [
+                Sight::make('image')->render(function (Donation $donation) {
+                    return "<img src={$donation['image']}
+                          alt='sample'
```
Comment (rubenderigo) [inline app/Orchid/Screens/DonationEditScreen.php:91]: alt atribute could be about the picture?

File: app/Orchid/Screens/DonationEditScreen.php [hunk 2]
```diff
@@ -102,6 +107,11 @@ public function layout(): array
                     ->help('Cambia el estado de una donacion')
             ]),
             Layout::legend('donation.user', [
+                Sight::make('image')->render(function (User $user) {
+                    return "<img src={$user['image']}
+                          alt='sample'
```
Comment (rubenderigo) [inline app/Orchid/Screens/DonationEditScreen.php:112]: same the other

File: app/Orchid/Screens/DonationEditScreen.php [hunk 3]
```diff
@@ -88,7 +88,7 @@ public function layout(): array
              Layout::legend('donation', [
                 Sight::make('image')->render(function (Donation $donation) {
                     return "<img src={$donation['image']}
-                          alt='sample'
+                          alt='image-donation'
```
Comment (rubenderigo) [inline app/Orchid/Screens/DonationEditScreen.php:91]: is not necessary add "image" because the alt attribute belong to img etiquette (is redundant)

**Review by rubenderigo — APPROVED** [review]

**Review by rubenderigo — CHANGES_REQUESTED** [review]
missing alt attribute

**Review by rubenderigo — APPROVED** [review]
LGTM

**Pull request merged**